lib/Backend/Commands.hs

```diff
@@ -382,21 +385,23 @@ copyCmd
      , Member (Error CopyResult) r
      )
   => CopyOptions -> Sem r CopyResult
-copyCmd (CopyOptions oldPath newPath force) = do
+copyCmd (CopyOptions dryRun oldPath newPath force) = do
   operations <- buildCopyOperations oldPath newPath force
-  runCopyOperations operations
+  when (not dryRun) do
+    runCopyOperations operations
   pure $ CPRSuccess $ getOperationPaths <$> operations
 
 deleteCmd
   :: (Member BackendEffect r, Member (Error CofferError) r, Member (Error DeleteResult) r)
   => DeleteOptions -> Sem r DeleteResult
-deleteCmd (DeleteOptions path recursive) = do
+deleteCmd (DeleteOptions dryRun path recursive) = do
   getEntryOrDirThrow DRPathNotFound path >>= \case
-    Left entry -> deleteSecret (entry ^. E.path) $> DRSuccess [entry ^. E.path]
+    Left entry -> when (not dryRun) (deleteSecret (entry ^. E.path)) $> DRSuccess [entry ^. E.path]
```

Comment:
```suggestion
    Left entry -> unless dryRun (deleteSecret (entry ^. E.path)) $> DRSuccess [entry ^. E.path]
```

Please, don't combine `when`/`unless` into one-liners, prefer
```hs
when smth do
  smthElse

return other
```

Even when you're changing old code.

Reply:
Done